python/lsst/obs/hsc/makeTransmissionCurves.py

```diff
@@ -36,20 +35,23 @@
 
 HSC_BEGIN = "2012-12-18"  # initial date for curves valid for entire lifetime of HSC
 
+OLD_FILTER_NAME_MAP = {b: f"HSC-{b.upper()}" for b in "grizy"}
+OLD_FILTER_NAME_MAP.update(r2="HSC-R2", i2="HSC-I2")
 
-def getLongFilterName(short):
-    """Return a long HSC filter name (e.g. 'HSC-R') that's usable as a data ID
-    value from the short one (e.g. 'r') declared canonical in afw.image.Filter.
+
+def getPhysicalFilterName(short):
+    """Return am HSC physical_filter name (e.g. 'HSC-R') that's usable as a
+    data ID value from various names used in the transmission data files.
     """
     if short.startswith("HSC"):
         return short
     if short.startswith("NB") or short.startswith("IB"):
         num = int(short[2:].lstrip("0"))
         return "%s%04d" % (short[:2], num)
-    for filter in hscFilters.HSC_FILTER_DEFINITIONS:
-        if short == filter.afw_name or short == filter.band:
-            return filter.physical_filter
-    return short
+    if short.startswith("N"):
```

Comment:
Might be worth a comment above this that it's just here for the narrowband filters?